processor/src/tests/mod.rs

```diff
@@ -16,9 +16,7 @@ mod addresses;
 pub(crate) use addresses::test_addresses;
 
 // Effective Once
-lazy_static::lazy_static! {
-  static ref INIT_LOGGER: () = env_logger::init();
-}
+static INIT_LOGGER: once_cell::sync::Lazy<()> = once_cell::sync::Lazy::new(|| env_logger::init);
```

Comment:
I think this should actually be written as `::new(env_logger::init)`, not `::new(|| env_logger::init())`. clippy will likely complain otherwise.

Reply:
Sorry for all the pushes, I don't have librocksdb so can't test locally :)

The last push failed because clippy wanted the function item directly, it wanted `::new(env_logger::init)` instead of `::new(|| env_logger::init())`, I just forgot to get rid of the closure.